### ISSUE TITLE
[Bug]: Netdata autoupdate generates ipv4_tcp_resets

### ISSUE THREAD

**Issue by UltimateByte** [netdata/netdata#13742]
### Bug description

Most mornings at relatively fixed times, I saw ipv4_tcp_resets email alerts on most of my servers, that went "Clear" right after.

After digging syslogs, it appears to occur during netdata updates.
Examples:
![unknown](https://user-images.githubusercontent.com/8805941/192993628-7fca415f-719a-4107-8658-09e747b8050a.png)
```
Sep 29 06:25:01 moon CRON[287730]: (root) CMD (test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.daily ))
Sep 29 06:25:44 moon systemd[1]: Reloading.
Sep 29 06:25:44 moon systemd[1]: Starting Daily apt upgrade and clean activities...
Sep 29 06:25:44 moon systemd[1]: Stopping Real time performance monitoring...
Sep 29 06:25:44 moon systemd[1]: apt-daily-upgrade.service: Succeeded.
Sep 29 06:25:44 moon systemd[1]: Finished Daily apt upgrade and clean activities.
Sep 29 06:25:48 moon systemd[1]: netdata.service: Succeeded.
Sep 29 06:25:48 moon systemd[1]: Stopped Real time performance monitoring.
Sep 29 06:25:48 moon systemd[1]: netdata.service: Consumed 1h 21min 5.280s CPU time.
Sep 29 06:25:48 moon systemd[1]: Started Real time performance monitoring.
```
![lobby](https://user-images.githubusercontent.com/8805941/192993780-102f51bb-eb6c-4bfd-b54a-027fbde75d15.png)
```
Sep 29 06:51:43 lobby systemd[1]: Reloading.
Sep 29 06:51:43 lobby systemd[1]: Stopping Real time performance monitoring...
Sep 29 06:51:47 lobby systemd[1]: netdata.service: Succeeded.
Sep 29 06:51:47 lobby systemd[1]: Stopped Real time performance monitoring.
Sep 29 06:51:47 lobby systemd[1]: netdata.service: Consumed 51min 36.310s CPU time.
Sep 29 06:51:47 lobby systemd[1]: Started Real time performance monitoring.
```

Correlation does not always mean causality, however in this case, it seems to be precisely linked on all servers that I've checked.

It may only occur on servers with low traffic, as errors then represent a high percentage of traffic.

### Expected behavior

Updating Netdata should not trigger an ipv4_tcp_resets alert.

### Steps to reproduce

1. Use Netdata on Debian 11 based installs (native packages) 
2. Have your nodes claimed on Netdata Cloud, therefore receive alerts
3. Have low traffic when auto-updates occur
4. Receive email alerts
5. Check in your syslog is these alerts happened at the same time as auto-updates, if yes, then it's successfully reproduced.

### Installation method

kickstart.sh

### System info

```shell
Linux moon.lrob.net 5.10.0-18-amd64 #1 SMP Debian 5.10.140-1 (2022-09-02) x86_64 GNU/Linux
/etc/os-release:PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
/etc/os-release:NAME="Debian GNU/Linux"
/etc/os-release:VERSION_ID="11"
/etc/os-release:VERSION="11 (bullseye)"
/etc/os-release:VERSION_CODENAME=bullseye
/etc/os-release:ID=debian
```

### Netdata build info

```shell
root@moon:~# $(ps aux | grep -m1 -E -o "[a-zA-Z/]+netdata ") -W buildinfo
Version: netdata v1.36.0-158-nightly
Configure options:  '--build=x86_64-linux-gnu' '--includedir=${prefix}/include' '--mandir=${prefix}/share/man' '--infodir=${prefix}/share/info' '--disable-option-checking' '--disable-silent-rules' '--libdir=${prefix}/lib/x86_64-linux-gnu' '--libexecdir=${prefix}/lib/x86_64-linux-gnu' '--disable-maintainer-mode' '--prefix=/usr' '--sysconfdir=/etc' '--localstatedir=/var' '--libdir=/usr/lib' '--libexecdir=/usr/libexec' '--with-user=netdata' '--with-math' '--with-zlib' '--with-webdir=/var/lib/netdata/www' '--disable-dependency-tracking' 'build_alias=x86_64-linux-gnu' 'CFLAGS=-g -O2 -ffile-prefix-map=/usr/src/netdata=. -fstack-protector-strong -Wformat -Werror=format-security' 'LDFLAGS=-Wl,-z,relro' 'CPPFLAGS=-Wdate-time -D_FORTIFY_SOURCE=2' 'CXXFLAGS=-g -O2 -ffile-prefix-map=/usr/src/netdata=. -fstack-protector-strong -Wformat -Werror=format-security'
Install type: binpkg-deb
    Binary architecture: x86_64
    Packaging distro:
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES
    ACLK:                       YES
    TLS Host Verification:      YES
    Machine Learning:           YES
    Stream Compression:         YES
Libraries:
    protobuf:                YES (system)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  NO
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    YES
    EBPF:                    YES
    IPMI:                    YES
    NFACCT:                  YES
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 NO
    Prometheus Remote Write: YES
```

**Comment by ilyam8** [comment]
Hi, @UltimateByte. Can you try the changing the warn

https://github.com/netdata/netdata/blob/22cc4d60d1af3a5e06e600307d50fc99d5409056/health/health.d/tcp_resets.conf#L19-L29

to

```bash
$netdata.uptime.uptime > (1 * 60) AND $this > ((($1m_ipv4_tcp_resets_sent < 5)?(5):($1m_ipv4_tcp_resets_sent)) * (($status >= $WARNING)  ? (1) : (10)))
```

I add `netdata.uptime.uptime > (1 * 60) AND`, it adds 1 minute delay after Netdata (re)start. The `10s_ipv4_tcp_resets_received` can be modified in the same way.

**Comment by UltimateByte** [comment]
Hi @ilyam8 
I've applied the change on 3 servers with low traffic. Now we wait for the next update I assume... ? 😄

**Comment by ilyam8** [comment]
Yes.

**Comment by ilyam8** [comment]
@UltimateByte hey, did the patch help?

**Comment by UltimateByte** [comment]
Hello @ilyam8 
I now can confirm that I didn't receive any alert from the nodes with altered config, while they did run auto-updates.
Though I can't be 100% sure it is due to the fix, since the alert is not triggered 100% of the time, chances are pretty high IMHO that the config change helped since almost every other node triggered alarms.

Planning to implement this by default? It probably could and should be. :)

**Comment by cakrit** [comment]
Netdata shouldn't be generating any alerts due to its update process.

**Comment by UltimateByte** [comment]
@cakrit That's actually a relevant remark.
Maybe the solution doesn't lie in fixing the alert but rather in understanding the root cause for the TCP drops generated by the update. In the meantime, altering the alert config by default with the proposed fix would still be a suitable solution as far as I'm concerned.

**Comment by ilyam8** [comment]
- "understanding root cause" is not a blocker for adding a temporary (permanent) workaround.
- the problem is not new, but very old (see #9799).

The OP issue is (I hope) fixed in #13761.

**Comment by UltimateByte** [comment]
Perfect, thank you!
